packages/financial-templates-lib/src/price-feed/PriceFeedMockScaled.js

```diff
@@ -2,74 +2,89 @@ const { toBN } = require("web3").utils;
 const { PriceFeedInterface } = require("./PriceFeedInterface");
```

Comment:
This file has a lot of diffs because I basically copy and pasted the `PriceFeedMock` code here but added the final conversion step. This file is not currently used in any other tests.